Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -2006,23 +2078,17 @@ def main():
     username = params.get('credentials', {}).get('identifier')
     password = params.get('credentials', {}).get('password')
 
-    api_token = password if '_token' in username else None
-
-    # Remove trailing slash to prevent wrong URL path to service
-    server = params['url'].strip('/')
-    # Should we use SSL
-    use_ssl = not params.get('insecure', False)
-
     # Remove proxy if not set to true in params
     handle_proxy()
     command = demisto.command()
 
     client = Client(
-        server,
+        params.get('url').strip('/'),
         username,
         password,
-        api_token=api_token,
-        verify=use_ssl
+        api_token=password if '_token' in username else None,
+        verify=not params.get('insecure', False),
+        api_version=params.get('api_version', '4.x')
```

Comment:
Please switch to 3.x - if a user has a preconfigured instance, they won't have this param value - and if they used it until now, it means they're probably using 3.x.
```suggestion
        api_version=params.get('api_version', '3.x')
```